example/src/main/java/com/github/appintro/example/ui/custom/fragments/AnimatedDrawableFragment.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.appintro.example.ui.custom.fragments
+
+import android.graphics.drawable.AnimationDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.fragment.app.Fragment
+import com.github.appintro.example.R
+
+class AnimatedDrawableFragment : Fragment() {
+
+    private lateinit var myAnimation: AnimationDrawable
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.intro_animated_drawable, container, false)
```

Comment:
`R.layout.intro_animated_drawable` is probably not needed and you should be fine using the layouts provided by AppIntro.

Reply:
The problem is that the default layouts don't crop and fit the drawables correctly, you can try it with the default layouts, it looks weird.
If you don't have time, I can show you how it looks when I get back from school.

Reply:
> The problem is that the default layouts don't crop and fit the drawables correctly, you can try it with the default layouts, it looks weird.

This looks like a bug honestly. If so, we should fix it outside of this PR

Reply:
I would not call it a bug.
I think it's just because it's an animated drawable and it does not work for animated drawables.

Reply:
Ok, it's not a bug. The used drawables were just too small.
Replaced the old drawables with new ones.
Also now using the default intro layout.